packages/lookit-initjspsych/src/utils.spec.ts

```diff
@@ -1,60 +1,43 @@
 import { DataCollection } from "jspsych/dist/modules/data/DataCollection";
 
-import { ResponseData } from "./types";
-import { csrfToken, get, on_data_update, on_finish, patch } from "./utils";
+import { Child, PastSession, Study } from "@lookit/data/dist/types";
+import { on_data_update, on_finish } from "./utils";
 
 delete global.window.location;
 global.window = Object.create(window);
 global.window.location = { replace: jest.fn() };
 
-test("Find CSRF Token in cookies.", () => {
-  expect(csrfToken()).toBe("");
-
-  const token = "some-csrf-token";
-  document.cookie = `csrftoken=${token};`;
-
-  expect(csrfToken()).toBe(token);
-});
-
-test("Restful Get", async () => {
-  const jsonData = { date: new Date() };
-  const response = {
-    json: () => Promise.resolve(jsonData),
-    ok: true,
-  } as Response;
-  global.fetch = jest.fn(() => Promise.resolve(response));
-  global.Request = jest.fn();
-
-  expect(await get("some-url")).toBe(jsonData);
-  expect(fetch).toHaveBeenCalledTimes(1);
-  expect(Request).toHaveBeenCalledTimes(1);
-});
-
-test("Restful Patch", async () => {
-  const jsonData = { date: new Date() };
-  const expData = {} as DataCollection;
-  const data: ResponseData = {
-    id: "uuid id",
-    type: "responses",
-    attributes: { exp_data: [expData], completed: false },
-  };
+test("jsPsych's on_data_update", async () => {
```

Comment:
This test name is used twice - here and the next test. It looks like the only difference is that this one uses an empty array for `exp_data` and the next one uses `undefined`. 
Can we give unique names to these?